meeting.js

```diff
@@ -0,0 +1,330 @@
+    switch (media.type) {
+      case 'remoteVideo':
+        RemoteVideo.srcObject = media.stream;
+        break;
+      case 'remoteAudio':
+        RemoteAudio.srcObject = media.stream;
+        RemoteAudio.play();
+        break;
+      default:
+        console.error('Unknown media type:', media.type);
+    }
+  });
+  createdMeeting.on('media:stopped', (media) => {
+    switch (media.type) {
+      case 'remoteVideo':
+        RemoteVideo.srcObject = null;
+        break;
+      case 'remoteAudio':
+        RemoteAudio.srcObject = null;
+        break;
+      default:
+        console.error('Unknown media type:', media.type);
+    }
+  });
+}
+
+// 8) Get Local Streams Function.
+async function getLocalStreams() {
+  try {
+    const microphoneStream = await webex.meetings.mediaHelpers.createMicrophoneStream({
+      echoCancellation: true,
+      noiseSuppression: true,
+    });
+
+    const cameraStream = await webex.meetings.mediaHelpers.createCameraStream({ width: 640, height: 480 });
+    localVideo.srcObject = cameraStream.outputStream;
+
+    return {
+      microphone: microphoneStream,
+      camera: cameraStream,
+    };
+  } catch (error) {
+    console.error('Error getting local streams:', error);
+    throw error;
+  }
+}
+
+// 9) Joining the Meeting and add Local Streams to meeting.
+async function joinMeetingWithMedia(localStreams) {
+  try {
+    const meetingOptions = {
+      mediaOptions: {
+        allowMediaInLobby: true,
+        shareAudioEnabled: false,
+        shareVideoEnabled: false,
+        localStreams,      
+      },
+    };
+
+    await createdMeeting.joinWithMedia(meetingOptions);
+
+    document.getElementById('joinMeetingModal').style.display = 'block';
+    joinMeetingButton.style.display = 'none';
+  } catch (error) {
+    console.error('Error joining meeting with media:', error);
+    throw error;
+  }
+}
+
+// 10) Join Meeting Listener and Calling Above Functions(6 to 9).
+joinMeetingButton.addEventListener('click', joinMeeting);
+async function joinMeeting() {
+  document.getElementById("joinMeetingButton").style.display = "none";
+  document.getElementById("localvideoimage").style.display = "none";
+  $('#joinMeetingModal').modal('show');
+  var element = document.querySelector('.container3');
+  element.classList.add('blur');
+  try {
+    await createMeeting();
+    setMediaListeners();
+    localStream = await getLocalStreams();
+    createdMeeting.setRemoteQualityLevel('HIGH');
+    await joinMeetingWithMedia(localStream);
+  } catch (error) {
+    console.error('Error joining meeting:', error);
+  }
+}
+
+// 11) Leaving Meeting Listener and Function.
+leaveMeetingButton.addEventListener('click', leaveMeeting);
+async function leaveMeeting() {
+  try {
+    const confirmLeave = window.confirm("Are you sure you want to leave the meeting?");
+    if(confirmLeave){
+      await createdMeeting.leave();
+      joinMeetingButton.style.display = 'block';
+      $('#joinMeetingModal').modal('hide');

```

Comment:
Let us try to get rid of jquery and use Browsers DOM methods like `document.getElementById`

This comment is applicable wherever we have used jQuery `$()`

Reply:
This is modelview bootstrap command so when we use bootstrap command we use jquery

Reply:
That was by default modal-view format.When we use bootstrap modal view we should use jquerry to show the modal view

Reply:
By using DOM methods we cannot achive the modal view as you can see the method use modal method.

Reply:
Oh. So, the bootstrap framework mandates jquery usage? Got it.